src/MarginTrading.SettingsService.Services/LogToSql.cs

```diff
@@ -0,0 +1,123 @@
+        {
+            await WriteLog(LogLevel.Info, component, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteMonitorAsync(string component, string process, string context, string info, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Monitoring, component, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteWarningAsync(string component, string process, string context, string info, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Warning, component, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteWarningAsync(string component, string process, string context, string info, Exception ex,
+            DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Warning, component, process, context, info, ex, dateTime);
+        }
+
+        public async Task WriteErrorAsync(string component, string process, string context, Exception exception, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Error, component, process, context, null, exception, dateTime);
+        }
+
+        public async Task WriteFatalErrorAsync(string component, string process, string context, Exception exception,
+            DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.FatalError, component, process, context, null, exception, dateTime);
+        }
+
+        public async Task WriteInfoAsync(string process, string context, string info, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Info, null, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteMonitorAsync(string process, string context, string info, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Monitoring, null, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteWarningAsync(string process, string context, string info, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Warning, null, process, context, info, null, dateTime);
+        }
+
+        public async Task WriteWarningAsync(string process, string context, string info, Exception ex, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Warning, null, process, context, info, ex, dateTime);
+        }
+
+        public async Task WriteErrorAsync(string process, string context, Exception exception, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.Error, null, process, context, null, exception, dateTime);
+        }
+
+        public async Task WriteFatalErrorAsync(string process, string context, Exception exception, DateTime? dateTime = null)
+        {
+            await WriteLog(LogLevel.FatalError, null, process, context, null, exception, dateTime);
+        }
+        
+        private static string Truncate(string str)

```

Comment:
It is not necessary in SQL, but not very important for now.

Reply:
Sure, just cutting off overabundant data